### DISCUSSION TITLE
Debugging support tools

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#16422] [Feature improvement]
To be able to better find out whether an issue is a local issue (typically due to a configuration error or a customization) or a bug, I first turn to https://gitdeveloper.yetiforce.com to try to replicate the issue.

When I can replicate the issue, I submit a bug report, the "good" case.

More often than not, the developer (and test) environment do not behave like by installation.

Therefore it would be helpful to be able to:
1. see the `errors.log` file on `https://gitdeveloper.yetiforce.com`, maybe by adding a function `Logs / Show error logfile` in the administrator menu that display the error log file in reverse order
2. see the structure and/or contents of specific SQL tables (I have no idea yet how to do this easily).
in order to trace possible issues. 

Sometimes (most of the times), the error is on my side. Sometimes (probably due to legacy) I have incorrect values in the database that are responsible for the problem. As I cannot cross-check if the value is incorrect only with me or also with the "official" version, I cannot help with getting these issues fixed on a larger scale.

For example, I found that the `field_name` relating to an `OSSTimeControl (51)` entry for the `Contact (4)` table had as value `link`, rather than `linkexted`. I currently have no way of checking if that is the case in the "official" version (except maybe installing a second instance of the CRM system just to cross-check) or only at my site.
![issue](https://user-images.githubusercontent.com/10230144/187420610-d811d5a7-0a95-4c64-b6d9-3793a1c61ed4.jpg)

**Comment by rskrzypczak** [comment]
> I currently have no way of checking if that is the case in the "official" version (except maybe installing a second instance of the CRM system just to cross-check) or only at my site.

or you can check the .sql files directly (https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/install/install_schema) :)

As for the entry indicated by you, it simply does not exist. There is no Time Control tab on the contact.
We'll add it soon, as it should be, and so 'linkextend' is the correct value here.

1./2. You are probably misinterpreting the purposefulness of the demo environments we provide. For many reasons, no logs should be displayed there.
